recipes/ARIA-tools/build_base.sh

```diff
@@ -0,0 +1,2 @@
+export SETUPTOOLS_SCM_PRETEND_VERSION="$PKG_VERSION"
+$PYTHON -m pip install . -vv
```

Comment:
You don't need this file with the script line in the recipe.

Reply:
I am interpreting the comment as requesting to remove `build.sh`. Is that correct? Or should I remove one of the lines?

Thanks for your help with this.